app/middleware/acl.js

```diff
@@ -62,6 +62,14 @@ module.exports = async (req, res, next) => {
     await req.user.update({lastLoginAt: new Date()});
   }
 
+  try {
+    if (req.query.clinician_view && isAdmin(req.user)) {
+      req.user.groups = [{name: 'Clinician'}];
```

Comment:
is it possible to include managers as well as admins?

Reply:
Yes, fixing it